site/src/pages/components/popover-hint.research.explainer.mdx

```diff
@@ -0,0 +1,169 @@
+
+Finally, `popover=auto` popovers support "nesting": the idea that one popover is a descendant of another popover, for example in the case of nested menus. It makes sense in that example for the parent menu to stay open while the child menu opens. The same is not true for hints - it would be confusing for one hint to be nested inside another, and wouldn't make sense. Therefore, hints do not support nesting.
+
+
+# Accessibility
+
+A key distinction between types of hints is whether it may be important for a user to explore or interact with the hint. Consensus on these types of hints/tooltips can help with the resolution of [ARIA#979: Clarify the use of role=tooltip](https://github.com/w3c/aria/issues/979).
+
+- A plain hint contains only elements with a computed role of generic, text and images.
+- A rich hint contains other elements, such as links, form controls, lists or tables. These are hints that a screen reader user may need to interact with or explore.
+
+## Why differentiate between rich and plain hints?
+
+For a screen reader user, there already exists simple settings and commands to read accessible descriptions, which are just an additional piece of text associated with an object, such as used by a `title` or `aria-description` attribute.  When a hint is just used for a prettier tooltip, a screen reader can reuse this simple mechanism and act like a title was present. There is no need for the user to navigate to it.
+
+However, when a hint is used for something more complex, such as a dialog that appears on hover, or a data table, then the user needs to be able to learn it’s there and navigate to it, so that they can use additional screen reader commands to interact with it.
+
+## A11y API support
+
+For the best screen reading experience, the implementation will need to expose different properties for a plain hint vs a rich hint (classification described above).
+
+- Plain hints: the browser will simply expose the text on the triggering element. The actual popover target element and its descendants can be invisible/ignored in the AX tree.
+    1. If no other accessible name is available, use hint’s inner text for the accessible name
+    2. If it's used as the name, then use it to compute the description, setting the described by relation to point from the trigger to the hint element
+
+- Rich hints:
+    1. Add `aria-expanded` on the triggering element: the same treatment as for `popover=auto`.
+    2. Minimum role of `tooltip` on the target — see [spec PR describing minimum role](https://github.com/w3c/html-aam/pull/454).
+    3. Add `aria-details` pointing from the trigger to the popover. This allows screen readers to announce the presence of the popover and a command for navigating to it.
```

Comment:
we may want to overtly call out that a rich hint would not contribute to the description calculation, and i'm almost certain we should not include it in the naming calculation either.

Reply:
If you suggest some good wording for this, I'm happy to include it!

Reply:
per your mention of doing addon work in another issue/PR

Reply:
Great - thanks!